genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/v4/JobSpecification.java

```diff
@@ -90,6 +93,9 @@ public JobSpecification(
             : ImmutableMap.copyOf(environmentVariables);
         this.interactive = interactive;
         this.jobDirectoryLocation = jobDirectoryLocation;
+        this.archiveLocation = StringUtils.isBlank(archiveLocation)
```

Comment:
I'm not sure I understand the value of saving an empty string vs just saving the null? Also does a null represent that archiving is disabled?

Reply:
Will do